### ISSUE TITLE
[a11y] Stuttering effect in invisible texts as table captions

### ISSUE THREAD

**Issue by chmst** [joomla/joomla-cms#35279]
### Steps to reproduce the issue

Tables in Joomla have an invisible Caption with information about the content of the table. 
These texts are in general "Table of ...". 
Some screenreades announce table already as "table" and therefore read "table table of .. i.e. contacts". 

Replacing the Text by "List of .. " could solve the Problem.

The same could be for other elements, i.e. links or images - language files should be reviewed.

**Comment by brianteeman** [comment]
But then it might say "table of list of ..."

Links and images should all be ok as I did check those.

The new accessibility checker I am working on will also check links and images that are user created.

Personally I would not want to make a change to the captions without the specific recommendation of a regular screen reader user. There are lots of quirks with screen readers that users live with, expect, or have a plugin to check and its very very hard for a non screen reader user to evaluate this

**Comment by brianteeman** [comment]
rechecked with both Narrator and NVDA and neither of them have this problem

**Comment by richard67** [comment]
Closing as having a pull request. Please test #38761 . Thanks in advance.
